subsys/bluetooth/host/Kconfig

```diff
@@ -604,11 +604,11 @@ config BT_ECC
 	  This option adds support for ECDH HCI commands.
 
 config BT_TINYCRYPT_ECC
-	bool "Use TinyCrypt library for ECDH"
+	bool "Emulate ECDH in the Host using TinyCrypt library"
 	select TINYCRYPT
 	select TINYCRYPT_ECC_DH
 	depends on BT_ECC && (BT_HCI_RAW || BT_HCI_HOST)
-	default y if BT_CTLR
+	default y if BT_CTLR && !BT_CTLR_ECDH_SUPPORT
```

Comment:
Should be dependent on if the controller enables ECDH, not if the controller design supports it

Reply:
Good point!

Updated to add a features selection that is visible if controller supports the feature.